docs/composedb/guides/data-modeling/composites.mdx

```diff
@@ -33,7 +33,7 @@ Let’s say you have a model written in a `my-schema.graphql` file. To convert t
   <TabItem value="cli">
 
 ```bash
-composedb composite:create my-schema.graphql --output=my-composite.json --did-private-key=your-private-key
+composedb composite:create my-schema.graphql --output=./__generated__/definition.json --did-private-key=your-private-key
```

Comment:
This command does return a simple json file instead of a file inside the `__generated__` subdirectory, I mean it will create the output where the users specifies, but then they might confuse this with the definition json file used in the runtime when using the composedb server that is autogenerated by the `writeEncodedCompositeRuntime` function